docs/source/install.md

```diff
@@ -0,0 +1,188 @@
+# Install Fabric (& Samples)
+
+Please install the [Prerequisites](./prereqs.html) before you following these install instructions for Fabric and Samples.
+
+#### Mac
+
+> **Fabric:** Install Fabric in `$HOME/go/src/github.com/<your_github_userid>` directory.  This is a Golang Community recommendation for Go projects and this location will also enable local directories to be shared with Docker containers
+
+>> **Fork** the Upstream Repo. Navigate to [(https://github.com/hyperledger/fabric)](https://github.com/hyperledger/fabric) and fork the fabric repository using the fork button in the top-right corner. 
```

Comment:
This seems to mix the instructions for Fabric developers and for those who are getting started. I think the contributors section is a better place for the small number of users who are going to contribute to the project

Reply:
Agreed - we need to make sure its the right set of install work for a new developer and we also want to keep it consistent with how we have been doing things to date. This was also discussed in the documentation calls and taking everyone's feedback, I have redone the page - where we start with Fabric Samples and give people the option to download Fabric if they want but the install/make/run instructions will remain in the Contributor's page.

The idea behind giving developers (By developer I mean anyone interested in Fabric - not someone looking to Contribute necessarily) the option to download the Fabric repo was to build on the their interest in Fabric - they are here because they want to understand and use Fabric - so lets give them the option to have Fabric on their machines so they can delve into the code, check out implementation, etc.  But let's not complicated it with make or running integration tests instructions.  And yes, let's keep the Samples instructions on top for continuity.